Client/src/Pages/Infrastructure/CreateMonitor/index.jsx

```diff
@@ -311,12 +341,11 @@ const CreateInfrastructureMonitor = () => {
 								(notification) => notification.type === "email"
 							)}
 							value={user?.email}
-							onChange={(e) => handleChange(e)}
+							onChange={(event) => handleNotifications(event, "email")}
```

Comment:
We are calling on change with the previous params?

Reply:
> We are calling on change with the previous params?

Eventually we will have more notification types to deal with than "Email" so I added a `type` param to the `handleNotification` method signature:

```
const handleNotifications = (event, type) => {...}
```

This seemed like the easiest way of dealing with potentially different types of notifications :thinking:   I'm open to alternatives as always though!

Reply:
My bad, I misread the function call!